content/en/getting-started/camera-positioning.md

```diff
@@ -7,44 +7,34 @@ weight: 3
 
 {{< tabs "Install Tab" >}}
 {{< tab "Correct mount" >}}
-There are two different possibilities to mount your cameras, as shown in the following graphic. 
-
-The first opportunity is depicted in the left picture, where the first camera is positioned at 11, the second camera is located between 1 and 18, and the third camera is placed between 17 and 2.
-
-The second opportunity is illustrated in the right picture, with the first camera at 6, the second camera positioned between 12 and 5, and the third camera between 7 and 19.
-
-Please keep in mind that while other configurations may work, the software is primarily developed to function with the two given setups.
-
-For some examples, please check out the 'Example' tab with some pictures from our awesome community!
 ![How to mount cameras](/camera-mount/images/camera_mount_final.png)
 
 ![How to mount cameras](/camera-mount/images/camera_mount_side.png)
 
+![Cameramount](/camera-mount/images/Dartboard_Plasma.jpg)
```

Comment:
This picture shows the Design by Robbel90, not the one that Autodarts has in its docs under 3d-printing. Not sure if we want to intermix Robbel90's design with Lars design within the docs. This might lead to confusion. However, what we should do, is to include Robbel90's design as a link and perhaps even say that his design is recommended (for 10 Euros) at the top of 3d-printing? What do you think?